replica/replica.go

```diff
@@ -196,23 +215,9 @@ func (replica *Replica) HandleMessage(m Message) {
 			replica.options.Logger.Debugf("ignore message: resync timestamp=%v compared to now=%v", timestamp, now)
 			return
 		}
+		replica.p.HandleMessage(m.Message)
 	}
```

Comment:
```suggestion
	}
	
	// Make sure that the Process state gets saved. We do this here
	// because Resync cannot cause state changes, so there is no
	// reason to save after handling a Resync message.
	defer replica.p.Save()
```